src/components/Municipality/MapAndSearch/index.js

```diff
@@ -23,7 +24,7 @@ export const MapAndSearch = () => {
   }, [leaderboardSegments]);
 
   useEffect(() => {
-    if (municipality) {
+    if (municipality && isMunicipalityPage) {
```

Comment:
This leads to the searched municipality not being highlighted.

Reply:
true! thx. It's fixed